R/tribble_paste.R

```diff
@@ -5,7 +5,13 @@ globalVariables(".rs.readUiPref", "datapasta") #ignore this function in R CMD ch
 #' @return The parsed table text. Useful for testing.
 #' @export
 #'
-tribble_paste <- function(){
+tribble_paste <- function(x) {
+
+  # if data is provided, try to use that to generate the tribble_paste()
+  if(!missing(x)) {
+    clipr::write_clip(x)
+  }
+
```

Comment:
I think we should just be able to do clipboard_table <- x. Might be safest to do as.data.frame(x) or test if x is the right class.